src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -67,96 +65,73 @@ public static IDicomServerBuilder AddHostedServices(this IDicomServerBuilder ser
     }
 
     /// <summary>
-    /// Adds services for enabling a DICOM server.
+    /// Adds services for enabling a DICOMWeb API.
     /// </summary>
-    /// <param name="services">The services collection.</param>
+    /// <param name="dicomServerBuilder">The services collection.</param>
     /// <param name="configurationRoot">An optional configuration root object. This method uses the "DicomServer" section.</param>
-    /// <param name="configureAction">An optional delegate to set <see cref="DicomServerConfiguration"/> properties after values have been loaded from configuration.</param>
+    /// <param name="configureAction">An optional delegate to set <see cref="ApiConfiguration"/> properties after values have been loaded from configuration.</param>
     /// <returns>A <see cref="IDicomServerBuilder"/> object.</returns>
-    public static IDicomServerBuilder AddDicomServer(
-        this IServiceCollection services,
+    public static IDicomServerBuilder AddWebApi(
```

Comment:
I wonder if the builder should be closer to something like a state machine, where the API guides callers to add the various components necessary for the server to run, such as the metadata store, file store, etc?

Reply:
Hmm...  I could see where this would be useful. I think the hard part of that today is that we are targeting the implementations vs the types of stores. e.g. `AddSqlServer` and `AddBlobDataStores` vs `AddFileStore`.